docs/developer/attributes/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Attributes and types overview
+sidebar_label: Overview
+description: Overview of attributes, Product, and Content types in Saleor.
+---
+
+An attribute is a reusable field that can be assigned to a product or a variant, similar to fields in a CMS.
```

Comment:
**suggestion (non-blocking)**:

Given that this is the first paragraph in this sidebar section, I think we should extend the description of what attributes are.

Reply:
Added note (many users actually expect attributes to be done via code like strapi)
<img width="1078" alt="image" src="https://github.com/saleor/saleor-docs/assets/4006792/672e74fe-4c40-488c-97ee-66a416d064b9">